src/store/matchedAudiences.ts

```diff
@@ -1,17 +1,17 @@
 import { storage, timeStampInSecs } from '../utils';
 import { StorageKeys, MatchedAudience } from '../../types';
 
-class MatchedAudienceStore {
-  matchedAudiences: MatchedAudience[];
-  matchedAudienceIds: string[];
+export class MatchedAudienceStore {
+  private matchedAudiences: MatchedAudience[];
```

Comment:
This `private` will require any new polyfills for consumers of the lib?

Reply:
I suppose this is a TS type annotation which is striped on compilation/build time.